spec/services/scrapers/low_priority_spec.rb

```diff
@@ -7,9 +7,9 @@
     it 'add specs' do
```

Comment:
also rename this to describe what it does :-)
`    it 'enqueues the court case worker' do`